examples/aks/aks_cluster/variables.tf

```diff
@@ -27,6 +27,11 @@ variable "castai_api_token" {
   description = "CAST AI API token created in console.cast.ai API Access keys section"
 }
 
+variable "castai_grpc_url" {
+  type        = string
+  description = "CAST AI gRPC URL"
+}
+
```

Comment:
Probably need to provide default URL if this is a mandatory variable.

Reply:
Very good catch. Added